package.json

```diff
@@ -7,8 +7,17 @@
     "start": "node index.js",
     "test": "echo \"Error: no test specified\" && exit 1"
   },
+  "type": "module",
   "private": true,
   "dependencies": {
-    "ws": "^8.18.0"
+    "@dashevo/dashcore-lib": "^0.21.3",
+    "bs58check": "^4.0.0",
```

Comment:
This module is not used right now. And I think you can do address validation through the dashcore-lib actually. 

this is how we do it

<img width="875" alt="image" src="https://github.com/user-attachments/assets/bd48ec95-5f2f-4ad6-9229-74ebd30622fd">